examples/qiskit/0_Getting_Started.ipynb

```diff
@@ -0,0 +1,587 @@
+{
```

Comment:
<p>Super nit: Using "Now" to segue into Hybrid Jobs makes it sound like we're about to get into the topic</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/265/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>